src/main/scala/org/broadinstitute/dsde/rawls/model/UserInfo.scala

```diff
@@ -1,8 +1,20 @@
 package org.broadinstitute.dsde.rawls.model
 
+import com.google.api.client.auth.oauth2.Credential
 import spray.http.OAuth2BearerToken
 
 /**
  * Created by dvoet on 7/21/15.
  */
 case class UserInfo(userEmail: String, accessToken: OAuth2BearerToken, accessTokenExpiresIn: Long, userSubjectId: String)
+
+object UserInfo {
+  def buildFromTokens(credential: Credential): UserInfo = {
+    val expiresInSeconds: Long = Option(credential.getExpiresInSeconds).map(_.toLong).getOrElse(0)
+    // TODO: alternate expiration test
```

Comment:
@jmthibault79 should we make a ticket for this or do you have a better way to test expiration?